client/my-sites/site-settings/delete-site-warning-dialog/index.jsx

```diff
@@ -16,6 +16,10 @@ function DeleteSiteWarningDialog( { isVisible, p2HubP2Count, onClose } ) {
 					{ translate( 'Go to your site listing' ) }
 				</Button>
 			);
+		} else if ( isTrialSite ) {
+			<Button primary href={ purchasesRoot }>
+				{ translate( 'Cancel trial', { context: 'button label' } ) }
+			</Button>;
```

Comment:
Didn't see this button showing up, we probably need to wrap it with `button.push` here.

Reply:
Can you re-review please? Think you got in just between my pushes, sorry!